ams/server/src/main/java/com/netease/arctic/server/dashboard/FormatTableDescriptor.java

```diff
@@ -59,6 +60,10 @@ public interface FormatTableDescriptor {
   Pair<List<OptimizingProcessInfo>, Integer> getOptimizingProcessesInfo(
       AmoroTable<?> amoroTable, int limit, int offset);
 
+  /** Get the paged optimizing process detail information of the {@link AmoroTable}. */
+  List<OptimizingProcessDetailInfo> getOptimizingProcessDetailInfo(
```

Comment:
The method can be renamed to `getOptimingTaskInfos`, and the class `OptimizingProcessDetailInfo` renamed to `OptimizingTaskInfo`.

Reply:
Done.